src/uct/ib/mlx5/gga/gga_mlx5.c

```diff
@@ -72,20 +108,7 @@ ucs_config_field_t uct_gga_mlx5_iface_config_table[] = {
   {NULL}
 };
 
-static ucs_status_t
-uct_ib_mlx5_gga_md_query(uct_md_h uct_md, uct_md_attr_v2_t *md_attr)
-{
-    ucs_status_t status;
-
-    status = uct_ib_mlx5_devx_md_query(uct_md, md_attr);
-    if (status != UCS_OK) {
-        return status;
-    }
-
-    /* GGA always packs exported mkey */
-    md_attr->rkey_packed_size = md_attr->exported_mkey_packed_size;
-    return UCS_OK;
-}
+static pthread_mutex_t uct_gga_mlx5_rkeys_locker = PTHREAD_MUTEX_INITIALIZER;
```

Comment:
maybe use "_lock" or "_mutex" to conform with the codebase
Maybe store it in the component uct_gga_component?

Reply:
I also thought about it, but don't found much benefit, just more code related to new type/struct